### ISSUE TITLE
Labeler reports should show labeler categories

### ISSUE THREAD

**Issue by heikadog** [bluesky-social/social-app#3381]
**Is your feature request related to a problem? Please describe.**
The categories for the labelers I use (e.g. Aegis, XBlock Screenshot Labeler) don't always apply to the current provided selections. For instance, when I report something to the XBlock Screenshot Labeler, it doesn't make sense to have options for Spam/Anti-Social Behavior/etc. I usually select "Other" and describe what I'm reporting, which isn't the most elegant solution. It'd be better and more accessible if the categories assigned to the labeler were in the reporting flow instead.

**Describe the solution you'd like**
The reporting flow for third-party labelers should show their provided categories instead of the current workflow. To use the example from earlier, I should be able to directly report something to the XBlock Screenshot Labeler as a Twitter screenshot.

**Describe alternatives you've considered**

- This, but more tightly integrated with Ozone—in tandem with this, show the specific categories something was reported for in the backend moderation interface itself.

**Additional context**
Not sure how this would look with labelers with a lot of categories and categories with long descriptions.

**Comment by cooljeanius** [comment]
Yeah, for the AI mod labeler I want to be able to report as AI, for XBlock I want to be able to report as the proper service an image is a screenshot from, and so on
